synapse/federation/send_queue.py

```diff
@@ -257,10 +256,10 @@ def get_replication_rows(self, from_token, to_token, limit, federation_ack=None)
         )
 
         for (key, (dest, user_id)) in dest_user_ids:
-            rows.append((key, PRESENCE_TYPE, {
-                "destination": dest,
-                "state": self.presence_map[user_id].as_dict(),
-            }))
+            rows.append((key, PresenceRow(
```

Comment:
could you document the type (and meaning) of `rows` next to where it is initialised? I think it is changing from a `tuple(str, str, dict)` to a `tuple(str, BaseFederationRow)` ?